### PR TITLE
UIORGS-100 Change the input of the Interface name to required

### PR DESCRIPTION
**Pull request by Vladyslav-Velytskyi**

### CODE REVIEW

**Comment by aliaksei-chumakou** [comment]
@Vladyslav-Velytskyi Thanks for your effort! but some points could be looked into:
1. please look to solve license/cla issue - you have to read and accept it.
2. you can remove warning of eslint above if it's fixed, just to not attract extra attention to what is fixed already.
3. Usually we assign PR on the person who is responsible to merge it. I'm not sure if you're able to merge, but if so - please feel free to re-assign it to yourself.

File: src/interfaces/EditInterface/EditInterface.js [hunk 1]
```diff
@@ -20,7 +20,10 @@ import {
   TextField,
 } from '@folio/stripes/components';
 
-import { isURLValid } from '../../Utils/Validate';
+import {
+  isURLValid,
+  Required,
```
Comment (aliaksei-chumakou) [inline src/interfaces/EditInterface/EditInterface.js]: `Required` - is a quite obsolete code, I understand that it's hard to know it for you, but in any case - could you please use `validateRequired` from `@folio/stripes-acq-components` ? Thanks

File: src/interfaces/EditInterface/EditInterface.js [hunk 2]
```diff
@@ -110,6 +113,7 @@ class EditInterface extends Component {
                 <Field
                   label={<FormattedMessage id="ui-organizations.interface.name" />}
                   name="name"
+                  validate={[Required]}
```
Comment (aliaksei-chumakou) [inline src/interfaces/EditInterface/EditInterface.js]:
```suggestion
                  validate={validateRequired}
```

File: src/interfaces/EditInterface/EditInterface.js (hunk 1 above)
Comment (Vladyslav-Velytskyi) [reply src/interfaces/EditInterface/EditInterface.js]: Yep, thanks for the comment
I correctly understand that i must to make new commit with this changes?

File: src/interfaces/EditInterface/EditInterface.js (hunk 2 above)
Comment (Vladyslav-Velytskyi) [reply src/interfaces/EditInterface/EditInterface.js]: I haven't seen the similar syntax regarding required input fields and made like how it looks in this module. Thanks, i'll fix it

**Pull request merged**